plugins/system_upgrade.py

```diff
@@ -511,6 +511,34 @@ def run_reboot(self):
         reboot()
 
     def run_download(self):
+        #  downstream hack to narrow missing upgrade path with modules between dist versions
+        if self.opts.command not in ['offline-upgrade', 'offline-distrosync']:
+            if dnf.base.WITH_MODULES:
+                module_container = self.base._moduleContainer
+                enabled_module_names = set()
```

Comment:
I am really sorry. After removal of first check for enabled. The name of the set is misleading. "module_names" would be a better choice.

Reply:
Thanks, I have updated it.